src/js/admin.js

```diff
@@ -432,6 +432,45 @@ WebcomAdmin.fileUploader = function($) {
     .on('click', '.meta-file-delete', removeFile);
 };
 
+WebcomAdmin.collegesUpload = function($) {
```

Comment:
Is there any way to repurpose webcomAdmin.fileUploader() instead of creating custom college-specific upload logic?